devel/design/networking-api.md

```diff
@@ -0,0 +1,509 @@
+# Linux Networking API
+
+## OVERVIEW
+Linux networking has always been a major subsystem in the Linux ecosystem. Its functionality is used broadly across domains, from connecting a local host to the network up to a fully functional network device (router, switch, traffic shaper, etc).
```

Comment:
It would be better to wrap the long lines.

Reply:
Linux networking has -> Networking has

(Avoid repetition)

Reply:
Done (both)